src/App.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from 'react';
+import styled, { ThemeProvider, createGlobalStyle } from 'styled-components';
+
+// Contexts
+import { ThemeContext } from './contexts/ThemeContext';
+// Theme
+import { theme, THEME_VARIANTS } from './theme';
+// Components
+import TaskList from './containers/TaskList';
+import ThemeToggler from './components/ThemeToggler';
+
+
+
+function App() {
+    const [currentTheme] = useContext(ThemeContext);
+
+    return (
+        <ThemeProvider theme={theme}>
+            <AppContainerSC variant={currentTheme === THEME_VARIANTS.LIGHT}>
+                <GlobalStyle />
+                <ThemeToggler />
+                <TaskList />
+            </AppContainerSC>
+        </ThemeProvider>
+    );
+}
+
+const GlobalStyle = createGlobalStyle`
+  body {
+      margin: 0;
+      padding: 0;
+      font-family: 'Lato', sans-serif; 
+  }
+`;
+
+const AppContainerSC = styled.div`
+    background: ${({ theme, variant }) =>
+        variant ? theme.colors.colorWhite : theme.colors.grey.dark};
```

Comment:
- Need to organize theme in a way that will give the ability to avoid ternary inside styled-components
- It is possible to make some helper getter function that will help to avoid breaking of DRY rule (in current realization  you will see dozen styled components with for example repeated `props => props.theme.colors.blue.dark` code fragments)